CarLot/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace OOP3
+{
+
+    public class Program
+    {
+        // program driver, has main
+
+        public static void Main(String[] args)
+        {
+            CarLot jimmiesCars = new CarLot("A");
+            CarLot canesCars = new CarLot("B");
+            Car mazda = new Car("coupe", 4, 123, "Kia", "Forte", 12000);
+            Car bmw = new Car("sedan", 2, 345, "Hyundai", "SUV", 2450.67);
+            Truck f150 = new Truck(2, 567, "Benz", "Truck", 43560.34);
+            Truck f1674 = new Truck(3, 897, "Toyota", "truck", 34592.3);
+            jimmiesCars.Add(mazda);
+            jimmiesCars.Add(bmw);
+            canesCars.Add(f150);
+            canesCars.Add(f1674);
+            jimmiesCars.PrintInventory();
+            canesCars.PrintInventory();
+
+
+
+        }
+
+    }
+
+    public class CarLot
+    {
+        // represnets a carlot
+        public List<Vehicle> inventory = new List<Vehicle>();
+        public String name { get; set; }
+        public CarLot(string initialName)
+        { this.name = initialName; }
+
+        public void Add(Vehicle vehicle)
+        {
+            // add vehicles to inventory list .
+            inventory.Add(vehicle);
+        }
+
+
+        public void PrintInventory()
+        {
+            //For each carlot, print all the components, i.e., license number, make, model, price  in it.
+            //for Cars, print type and number of doors in addition to them.
+            //for trucks, print size of bed in addition to them.
+            //Lastly, print number of vehicles in each car lot.
+            foreach (var a in inventory)
```

Comment:
var hides the data type, i would rather you be more explicit while learning about interfaces
``` 
foreach (Vehicle a in inventory)
```